pkg/api/v1/api.proto

```diff
@@ -79,6 +80,18 @@ message DeploymentEvent {
   optional ReleaseTrainSource release_train_source = 3;
 }
 
+message LockPreventedDeploymentEvent {
+  enum LockType {
+    LOCK_TYPE_UNKNOWN = 0;
```

Comment:
Do we really need to have the `LOCK_TYPE_UNKNOWN`? I think we can cross it out and we should.

Reply:
It's standard practice in protobuf to always map 0 to a special "unknown" value (https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum). Unless you know exactly how your system is evolving in the future, it's a good idea to stick to that.

Reply:
Fair enough, though now I can think of many places where this practice needs to be applied.